app/shared/cards/ResourceCard.js

```diff
@@ -4,7 +4,7 @@ import Link from 'next/link';
 
 export default function ResourceCard({ title, description, link }) {
   return (
-    <Link href={link} className="h-full">
+    <Link href={link} id="candidate_content" className="h-full">
```

Comment:
`id` needs to be unique on the page. you can add that as a class but not id (this is reused on the same page multiple times